compiler/rustc_macros/src/type_foldable.rs

```diff
@@ -16,8 +16,29 @@ pub fn type_foldable_derive(mut s: synstructure::Structure<'_>) -> proc_macro2::
         let bindings = vi.bindings();
         vi.construct(|_, index| {
             let bind = &bindings[index];
-            quote! {
-                ::rustc_middle::ty::fold::ir::TypeFoldable::try_fold_with(#bind, __folder)?
+
+            // fix fields with #[type_foldable(fixed)]
```

Comment:
the meaning of `fix` or `fixed` is not clear to me from just the name. Maybe `identity` would be better?

Reply:
Yes, fair point.  Originally I considered `invariant` and `self`, didn't think either was clear enough so came up with `fixpoint` but thought it was a bit wordy.  I agree `identity` is probably best.